src/Microsoft.Data.SqlClient/netfx/tools/targets/GenerateAssemblyInfo.targets

```diff
@@ -13,7 +13,7 @@
             <AssemblyInfo Include="using System.Runtime.InteropServices%3B" />
             <AssemblyInfo Include="using System.Runtime.Versioning%3B%0A" />
 
-            <AssemblyInfo Include="[assembly: System.CLSCompliantAttribute(true)]" />
+            <AssemblyInfo Include="[assembly: System.CLSCompliant(true)]" />
```

Comment:
This file bugs me. Why do we have all the using lines if we are going to reference all the attributes via their fully qualified name? Also, none of the lines below need the Attribute suffix. Since you are updating it anyway, can you changes lines 17-28 to simply be:

            <AssemblyInfo Include="[assembly: ComVisible(false)]" />
            <AssemblyInfo Include="[assembly: TargetFramework(&quot;.NETFramework,Version=$(TargetFrameworkVersion)&quot;)]" />
            <AssemblyInfo Include="[assembly: SatelliteContractVersion(&quot;$(AssemblyVersion)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyTitle(&quot;$(AssemblyName)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyDescription(&quot;$(Description)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyCompany(&quot;$(Company)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyProduct(&quot;$(Product)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyCopyright(&quot;$(Copyright)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyVersion(&quot;$(AssemblyVersion)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyFileVersion(&quot;$(AssemblyFileVersion)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyInformationalVersion(&quot;$(Version)&quot;)]" />
            <AssemblyInfo Include="[assembly: AssemblyConfiguration(&quot;$(Configuration)&quot;)]" />

Reply:
Ok 👍